docs/pages/get-started/faq.md

```diff
@@ -410,83 +423,141 @@ Currently, messaging incurs no fee. As XMTP decentralizes, messaging between par
+1. A user starts with an Ethereum wallet and installs an XMTP-enabled app.
+
+2. The user creates an XMTP identity, which involves generating an identity key.
+
+3. For each app or device, an installation key is created and signed by the identity key.
+
+4. The installation key is used to create an MLS credential, which is signed by the Ethereum wallet.
 
-To learn more about invitation and message encryption, see [Invitation and message encryption with XMTP](/protocol/v2/invitation-and-message-encryption).
+5. The MLS credential contains the wallet address and a hash of the installation's signature verification key.
+
+6. For group chats, XMTP uses the MLS protocol to manage secure communication between multiple parties.
+
+For more details, see [Group chat concepts and protocols](/protocol/v3/group-chat#security-and-encryption) and [Identity in XMTP v3](/protocol/v3/identity)
+
+#### XMTP V2
+
+In V2, blockchain accounts sign and advertise a set of keys to start using XMTP. These keys are used to establish a shared secret between the blockchain accounts. The shared secret is then used to generate a key for encrypting an invitation, allowing the blockchain accounts to start exchanging messages. No third-party apps or relayers are involved in this process.
+
+For more details, see [Key generation and usage in XMTP v2](/protocol/v2/key-generation-and-usage) and [Invitation and message encryption with XMTP v2](/protocol/v2/invitation-and-message-encryption).
 
 ### Does each blockchain account address have a corresponding XMTP identity?
 
+Yes, each blockchain account address is associated with an XMTP identity.
+
+1. When a user first uses XMTP with a particular blockchain account, they create an "identity key" for that account.
+
+2. This identity key is used to sign "installation keys" for each app or device where the user uses XMTP.
+
+3. The installation keys are then used to create and sign MLS credentials, which are used for secure communication in individual and group chats.
+
+To learn more about identity in XMTP v3, see [Identity in XMTP v3](/protocol/v3/identity).
+
+#### XMTP V2
+
 Yes. Each blockchain account address is represented by an XMTP identity key. This identity key is a part of a key bundle that only the address that generated it can use to authenticate messages.
 
 To learn more about XMTP identity keys, see [Key generation and usage in XMTP](/protocol/v2/key-generation-and-usage).
 
 ### Why do apps built with XMTP require a user to sign with their blockchain account private keys each time they start a new messaging session?
 
-When a user starts a new messaging session, they must sign with their blockchain account private key to decrypt their XMTP key bundle. The key bundle is then used for invitation and message decryption.
+Users need to authenticate with their blockchain account when starting a new messaging session in this way:
```

Comment:
is this an accurate summary of how v3 works?

Reply:
Similar to above - it's close, but we've nixed the identity key.

First session: Wallet signature to claim inbox ID

Every session: Wallet signature to certify that the app-generated installation key is allowed to act on behalf of the inbox. (Or put another way, to add the installation key to the inbox log's set of authorized keys)

Reply:
Thank you for this guidance as well, @richardhuaaa! 

I've made the following updates based on the information you've shared: https://github.com/xmtp/docs-xmtp-org/pull/26/files#diff-0f5558e2ccc886a61098eac3e4020166d6c4dee87249fc7dcda8eba0f5bd1196R483

WDYT? 

No rush on any of these for today, of course! Perhaps you might be able to take a look on Monday? Thank you again!

Reply:
LGTM!